mailpoet/build.sh

```diff
@@ -52,6 +52,9 @@ COMPOSER_MIRROR_PATH_REPOS=1 ./tools/vendor/composer.phar install --no-dev --pre
 echo '[BUILD] Fetching prefixed production libraries'
 ./tools/vendor/composer.phar install --no-dev --prefer-dist --working-dir=./prefixer/
 
+echo '[BUILD] Install email editor dependencies'
+cd ../packages/php/email-editor && ../../../mailpoet/tools/vendor/composer.phar install --no-dev --prefer-dist --optimize-autoloader --no-scripts && cd -
```

Comment:
The package dependencies are installed too late. I think they should be installed before the dependencies in line 50. I tried to check the build from CI, and the vendor directory for the `mailpoet` package was missing in the vendor.